trick_source/sim_services/Executive/Executive_init.cpp

```diff
@@ -33,16 +38,30 @@ int Trick::Executive::init() {
         struct rusage cpu_usage_buf ;
         getrusage(RUSAGE_SELF, &cpu_usage_buf);
         cpu_start =   ((double) cpu_usage_buf.ru_utime.tv_sec) + ((double) cpu_usage_buf.ru_utime.tv_usec / 1000000.0);
+        std::ofstream init_log_stream;
+
+        /* First parse command line to see if trick is in Sie generation mode. 
```

Comment:
I think this would be cleaner if we called process_sim_args here and handled the sie flag in that routine.  We should remove the call to sched.process_sim_args() in share/trick/sim_objects/default_trick_sys.sm when we move it here.

If init_log_stream is processed/initialized in process_sim_args, then for cleaner code perhaps init_log_stream should be part of the class, and not passed in as an argument.

Reply:
@alexlin0 I have implemented this, please take another look